src/yarpWholeBodySensors.cpp

```diff
@@ -621,6 +910,30 @@ bool yarpWholeBodySensors::openTorqueSensor(const int bp)
     return true;
 }
 
+vector< std::string > yarpWholeBodySensors::getSensorNames(const SensorType st)
+{
+    vector<std::string> ret;
+    IDList selectedSensors= sensorIdList[st];
+    for(int i=0; i < (int)sensorIdList[st].size(); i++)
+    {
+        switch(st)
+        {
+            case SENSOR_ACCELEROMETER : 
+                    ret.push_back(accelerometersReferenceIndeces[i].sensor_urdf_name);
+                break;
+            case SENSOR_GYROSCOPE : 
+                    ret.push_back(gyroscopesReferenceIndeces[i].sensor_urdf_name);
+                break;
+                
+            default : 
+                std::cerr<<" To be implemented for all kinds of sensors\n";
```

Comment:
Try to add the method (`yarpWholeBodySensors::getSensorNames`) in the error message, otherwise the user will get just the error: 
~~~
To be implemented for all kinds of sensors
~~~
with no clue on the origin.